cmake/CheckCopyright.cmake

```diff
@@ -14,10 +14,10 @@
 # TODO: Probably shouldn't be done at configure time - instead we should have a separate
 # build task that does this.
 
-file(GLOB_RECURSE COPYRIGHTABLE_SOURCES src/*.cpp example/*.cpp)
-file(GLOB_RECURSE COPYRIGHTABLE_HEADERS src/*.hpp example/*.hpp)
-file(GLOB_RECURSE COPYRIGHTABLE_SCRIPTS scripts/python/*.py)
-file(GLOB_RECURSE COPYRIGHTABLE_CMAKE cmake/* src/CMakeLists.txt example/CMakeLists.txt)
+file(GLOB_RECURSE COPYRIGHTABLE_SOURCES src/[^\.]*.cpp example/[^\.]*.cpp)
+file(GLOB_RECURSE COPYRIGHTABLE_HEADERS src/[^\.]*.hpp example/[^\.]*.hpp)
+file(GLOB_RECURSE COPYRIGHTABLE_SCRIPTS scripts/python/[^\.]*.py)
+file(GLOB_RECURSE COPYRIGHTABLE_CMAKE cmake/[^\.]* src/CMakeLists.txt example/CMakeLists.txt)
```

Comment:
The ".cpp" should actually be "\.cpp" and ditto for .hpp since the "." by itself will match any character.

Reply:
Never mind, I stand corrected.  Glob isn't the same as regex.
Thanks @AndrewGaspar